docs/operational_guide/replication_and_deployment_in_zones.md

```diff
@@ -0,0 +1,57 @@
+# Replication and Deployment in Zones
+
+## Overview
+
+M3DB supports deployment to both a single zone, with rack level isolation, and deployment across multiple zones in a region. You can also deploy M3DB across multiple regions for a global view of data, however you may experience higher ingestion latency and cross region bandwidth costs. You may want to use a replication factor of five for such a multi-region cluster to survive multiple regions suffering an outage, or be able to still upgrade hosts while a region is down.
+
+### Replication
+
+It is highly recommended that you use a replication factor of at least 3 for any M3DB deployment. This is due to the default write and read consistency levels requiring a majority of hosts to be available to complete an operation successfully. For more information see the documentation concerning tuning [availability, consistency and durability](availability_consistency_durability.md).
+
+### Upgrading hosts in a deployment
```

Comment:
I think this could use a little more detail to be clearer.

"When an M3DB node is restarted it has to go through a "bootstrapping" process to restore itself to a valid state. During this time the node will continue to accept writes, but will not be available for reads.

Obviously, there is also a small window of time during between when the process is stopped and then started again where it will also be unavailable for writes."

Reply:
Sure thing, thanks for the suggestion - looks good. I tweaked this just slightly.  It's not really restoring to a "valid state", it's just preparing to serve reads.